src/utils.c

```diff
@@ -2182,3 +2182,18 @@ void utils_start_new_geany_instance(const gchar *doc_path)
 	else
 		g_printerr("Unable to find 'geany'");
 }
+
+
+void utils_slist_free_full(GSList *list, GDestroyNotify deleter)
+{
+#if GLIB_CHECK_VERSION(2, 28, 0)
```

Comment:
oh, I guess it was missed when updating the version.  Actually we wanted to depend on GLib 2.28, we don't care about the GIO version so much (but it's the same)